input_data.go

```diff
@@ -1,146 +1,264 @@
-	Seven        Key = 55
-	Eight        Key = 56
-	Nine         Key = 57
-	F1           Key = 112
-	F2           Key = 113
-	F3           Key = 114
-	F4           Key = 115
-	F5           Key = 116
-	F6           Key = 117
-	F7           Key = 118
-	F8           Key = 119
-	F9           Key = 120
-	F10          Key = 121
-	F11          Key = 122
-	F12          Key = 123
-	A            Key = 65
-	B            Key = 66
-	C            Key = 67
-	D            Key = 68
-	E            Key = 69
-	F            Key = 70
-	G            Key = 71
-	H            Key = 72
-	I            Key = 73
-	J            Key = 74
-	K            Key = 75
-	L            Key = 76
-	M            Key = 77
-	N            Key = 78
-	O            Key = 79
-	P            Key = 80
-	Q            Key = 81
-	R            Key = 82
-	S            Key = 83
-	T            Key = 84
-	U            Key = 85
-	V            Key = 86
-	W            Key = 87
-	X            Key = 88
-	Y            Key = 89
-	Z            Key = 90
-	NumLock      Key = 144
-	NumMultiply  Key = 106
-	NumDivide    Key = 111
-	NumAdd       Key = 107
-	NumSubtract  Key = 109
-	NumZero      Key = 96
-	NumOne       Key = 97
-	NumTwo       Key = 98
-	NumThree     Key = 99
-	NumFour      Key = 100
-	NumFive      Key = 101
-	NumSix       Key = 102
-	NumSeven     Key = 103
-	NumEight     Key = 104
-	NumNine      Key = 105
-	NumDecimal   Key = 110
-	NumEnter     Key = 13
+	// RightSuper represents the right super keyboard key
+	// (Windows key on Microsoft Windows, Command key on Apple OSX, and varies on Linux)
+	RightSuper Key = 73
+	// RightAlt represents the left alt keyboard key
+	RightAlt Key = 18
+	// Zero represents the zero keyboard key '0'
+	Zero Key = 48
+	// One represents the one keyboard key '1'
+	One Key = 49
+	// Two represents the two keyboard key '2'
+	Two Key = 50
+	// Three represents the three keyboard key '3'
+	Three Key = 51
+	// Four represents the four keyboard key '4'
+	Four Key = 52
+	// Five represents the five keyboard key '5'
+	Five Key = 53
+	// Six represents the six keyboard key '6'
+	Six Key = 54
+	// Seven represents the seven keyboard key '7'
+	Seven Key = 55
+	// Eight represents the eight keyboard key '8'
+	Eight Key = 56
+	// Nine represents the nine keyboard key '9'
+	Nine Key = 57
+	// F1 represents the f1 keyboard key
+	F1 Key = 112
+	// F2 represents the f2 keyboard key
+	F2 Key = 113
+	// F3 represents the f3 keyboard key
+	F3 Key = 114
+	// F4 represents the f4 keyboard key
+	F4 Key = 115
+	// F5 represents the f5 keyboard key
+	F5 Key = 116
+	// F6 represents the f6 keyboard key
+	F6 Key = 117
+	// F7 represents the f7 keyboard key
+	F7 Key = 118
+	// F8 represents the f8 keyboard key
+	F8 Key = 119
+	// F9 represents the f9 keyboard key
+	F9 Key = 120
+	// F10 represents the f10 keyboard key
+	F10 Key = 121
+	// F11 represents the f11 keyboard key
+	F11 Key = 122
+	// F12 represents the f12 keyboard key
+	F12 Key = 123
+	// A represents the a keyboard key 'A'
```

Comment:
I don't think you need the letter in quotation marks at the end of the line.

It makes more sense to format it like the following:

> /key/ represents the '/key/' on the keyboard

Example:

> **A represents the 'a' key on the keyboard**